plugins/org.eclipse.fordiac.ide.contracts/src/org/eclipse/fordiac/ide/contracts/DefineFBDecisionTwoPinDialog.java

```diff
@@ -10,94 +10,40 @@
  * Contributors:
```

Comment:
you can/should also update the first line:
FROM  * Copyright (c) 2023 Paul Pavlicek 
TO * Copyright (c) 2023, 2024 Paul Pavlicek and others

:)